### PR TITLE
Add Gittyup Gui

### PR DESCRIPTION
**Pull request by Murmele**
## Changes

Adds Gittyup as a new GUI

### CODE REVIEW

File: resources/guis.yml [hunk 1]
```diff
@@ -218,6 +218,16 @@ guis:
   price: Free
   license: MIT
   order: 13
+- name: Gittyup
+  url: https://github.com/Murmele/Gittyup
```
Comment (Murmele) [inline resources/guis.yml:222]:
```suggestion
  url: https://murmele.github.io/Gittyup/
```

**Comment by pedrorijo91** [comment]
thanks @Murmele . let us know when the PR is ready to review (currently as Draft)

**Comment by ju674** [comment]
😔

**Comment by Murmele** [comment]
Hi @pedrorijo91, the PR is ready to review. Thank you

File: resources/guis.yml [hunk 2]
```diff
@@ -218,6 +218,16 @@ guis:
   price: Free
   license: MIT
   order: 13
+- name: Gittyup
+  url: https://murmele.github.io/Gittyup/
+  image_tag: guis/gittyup@2x.png
+  platforms:
+  - Windows
+  #- Mac
```
Comment (pedrorijo91) [inline resources/guis.yml:226]: remove the comment?

Comment (Murmele) [reply resources/guis.yml]: I removed it

**Comment by pedrorijo91** [comment]
thanks a lot @Murmele ! :)

**Comment by pedrorijo91** [comment]
@peff  i think you may need to reconect heroku and your github due to the issue they had recently

imagem

**Comment by pedrorijo91** [comment]
friendly ping @peff  ^

**Comment by ttaylorr** [comment]
@pedrorijo91 I've been poking at this from the Heroku side of things, and I'm also unable to reconnect our automated deploys, and it appears that Heroku has [temporarily disabled them](https://status.heroku.com/incidents/2413). I'll deploy this manually in the meantime.

**Comment by Murmele** [comment]
@ttaylorr thank you. I will create soon a new pullrequest, because we got also the mac build working.

**Pull request merged**